packages/react/src/components/form/Checkbox/Checkbox.module.css

```diff
@@ -18,93 +56,71 @@
   cursor: pointer;
 }
 
-/* Checkbox */
-.label::before {
-  content: '';
-  width: var(--fds-checkbox-size);
-  height: var(--fds-checkbox-size);
-  box-shadow: inset 0 0 0 2px var(--fds-checkbox-border-color);
-  background: var(--fds-checkbox-background);
-  flex-shrink: 0;
-  margin-inline: 6px;
-  margin-block: 6px;
-  border-radius: var(--fds-border_radius-medium);
-}
-
 .description {
-  padding-left: calc(var(--fds-checkbox-size) + 12px + var(--fds-spacing-1));
-  margin-top: calc(var(--fds-spacing-2) * -1);
+  margin-top: calc(var(--fds-spacing-3) * -1);
   color: var(--fds-semantic-text-neutral-subtle);
-}
-
-.input {
-  position: absolute;
-  width: 2.75rem;
-  height: 2.75rem;
-  z-index: 1;
-  opacity: 0;
-  cursor: pointer;
-  margin: 0;
+  grid-column: 2;
 }
 
 .disabled > .input,
 .disabled > .label,
-.disabled > .label::before {
+.disabled > .input::before {
   cursor: not-allowed;
 }
 
 .disabled > .label,
-.disabled > .label::before,
+.disabled > .input,
 .disabled > .description {
   opacity: var(--fds-opacity-disabled);
 }
 
-.input:focus-visible + .label::before {
+.readonly > .label,
+.readonly > .input,
+.readonly > .input::before {
+  cursor: default;
+}
+
+.input:focus-visible {
   outline: var(--fds-checkbox-focus-border-width) solid var(--fds-semantic-border-focus-outline);
```

Comment:
You probably need to also add `outline-offset: 0;` to get the correct focus style now with the styling changes to use `input`. 

In this PR:
![image](https://github.com/digdir/designsystemet/assets/1070981/45297f8a-81c8-431e-ac2f-24ee5307860f)

Design/Current:
![image](https://github.com/digdir/designsystemet/assets/1070981/6cef7750-5d2e-40bc-85ab-6619e72142d4)

Reply:
<img width="150px" src="https://github.com/digdir/designsystemet/assets/3340349/35ed4565-2bdf-4252-8444-26b0f5051a28"/>
<img width="150px" src="https://github.com/digdir/designsystemet/assets/3340349/5ed53470-e96b-4b75-8bdd-a08e8c229dcb">

Forgot to test the state cross browser. 😞 
I'll add the fix now

Reply:
ah, yeah, its easy to forget test other browsers 😅